### ISSUE TITLE
Encode HTML-Entities in Metadata

### ISSUE THREAD

**Issue by adiherzog** [scenarioo/scenarioo#104]
HTML-Entities in Metadata have to be encoded, e.g. < has to become &amp;lt; Otherwise, they are interpreted as HTML brackets.

**Comment by bruderol** [comment]
@adiherzog did you really solve this? Probably not, probably this was just a commit with a wrong issue referenced (because on the fork there where other issue numbers)

**Comment by adiherzog** [comment]
Yeah, this refers to https://github.com/MyrionPhoenixmoon/scenarioo/issues/104

Funny effect :-) Could be a bug or a feature of GitHub, who knows...

**Comment by bruderol** [comment]
Currently it is as follows: 
If the user fills in HTML in any string field, it will be rendered in the Viewer as HTML, otherwise he has to escape it. I think that this behaviour is fair enough, and we should just document it. 

Also the libraries might need to provide some escape helper functionality or even automatic escaping of some fields (e.g. ScenariooJS should escape error messages, filled in from the jasmine reporter, since this errors sometimes can contain HTML snippets that are not expected to be rendered).

Later we should also support to add Markdown (and render it!) .. then the user could use following simple syntax to ensure that any content is escaped:
```
value = "```\n<any-content>\n```"
```
